pkg/cmd/controller/image.go

```diff
@@ -81,8 +81,8 @@ func RunImageTriggerController(ctx *ControllerContext) (bool, error) {
 	})
 	sources = append(sources, imagetriggercontroller.TriggerSource{
 		Resource:  schema.GroupResource{Group: "batch", Resource: "cronjobs"},
-		Informer:  ctx.KubernetesInformers.Batch().V1beta1().CronJobs().Informer(),
-		Store:     ctx.KubernetesInformers.Batch().V1beta1().CronJobs().Informer().GetIndexer(),
+		Informer:  ctx.KubernetesInformers.Batch().V1().CronJobs().Informer(),
```

Comment:
@dmage @ricardomaraschini @openshift/openshift-team-build-api 

CronJob is GA in v1.21 - updating image change trigger sources.

Reply:
also bump @soltysh